packages/js/src/components/modals/UpsellModal.js

```diff
@@ -69,9 +69,9 @@ export const UpsellModal = ( {
 						<div className="yst-mx-auto">{ __( "BLACK FRIDAY | 30% OFF", "wordpress-seo" ) }</div>
 					</div> }
 					<div className="yst-py-6 yst-px-12">
-						<Title as="h3" className="yst-mb-1 yst-leading-5 yst-text-sm yst-font-normal">{ title }</Title>
+						<Title as="h3" className="yst-mb-1 yst-leading-5 yst-text-sm yst-font-medium">{ title }</Title>
```

Comment:
I changed the font to medium size for all upsell modals inside the editor. This is to be inline with [the Figma file](https://www.figma.com/design/A8L1q0TQfj7hAfqoezQxjI/Ads---WordPress?node-id=1-3&p=f&t=ZSYquLvKJOlw8Uqk-0)